content/metadata.yaml

```diff
@@ -53,3 +53,16 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2020-03-23
+    github: nilswellhausen
+    name: Nils Wellhausen
+    initials: NW
+    orcid: 0000-0001-8955-7582
+    email: nilsw@pennmedicine.upenn.edu
+    contributions:
+      - Writing
+      - Visualization
```

Comment:
Awesome that you want to do visualization! For now, just write the contributions you have submitted PRs for (so e.g., if you are proposing edits to the paper structure, then add yourself for one or both of the writing categories [here](https://casrai.org/credit/)).

Reply:
Hi rando2! I submitted a pull request to add a section about neutralizing antibodies in the treatment section. It could be either under antivirals or as its own section. Comments about that are much appreciated!